test/basedriver/helpers-specs.js

```diff
@@ -15,4 +23,20 @@ describe('helpers', () => {
       isPackageOrBundle('/path/to/an.apk').should.be.false;
     });
   });
+
+  describe('#unzipFile', () => {
+    it('should unzip a .zip file', async () => {
+      const appContents = await fs.readFile(getFixture('FakeIOSApp.app'));
+      await unzipFile(getFixture('FakeIOSApp.app.zip'));
+      await fs.readFile(getFixture('FakeIOSApp.app')).should.eventually.deep.equal(appContents);
```

Comment:
I don't understand what this is asserting. Is seems like it is checking that the `FakeIOSApp.app` didn't change? Is the test just making sure `unzipFile` doesn't error out? If so, it should assert that so it is clear.